ghcjs-dom-javascript/src/GHCJS/DOM/JSFFI/XMLHttpRequest.hs

```diff
@@ -32,7 +32,7 @@ throwXHRError 0 = return ()
 throwXHRError 1 = throwIO XHRAborted
 throwXHRError 2 = throwIO XHRError
 
-foreign import javascript interruptible "h$dom$sendXHR($1, $2, $c);" js_send :: XMLHttpRequest -> JSVal -> IO Int
+foreign import javascript interruptible "((x, y, c) => { h$dom$sendXHR(x, y, c); } )" js_send :: XMLHttpRequest -> JSVal -> IO Int
```

Comment:
The correct form with this syntax would be with an added `return` to return the `Int`:
```
foreign import javascript interruptible "((x, y, c) => { return h$dom$sendXHR(x, y, c); } )" js_send :: XMLHttpRequest -> JSVal -> IO Int
```
But then you can avoid the anonymous lambda with just:

```suggestion
foreign import javascript interruptible "h$dom$sendXHR" js_send :: XMLHttpRequest -> JSVal -> IO Int
```